neo4j/driver_examples_test.go

```diff
@@ -138,6 +143,32 @@ func ExampleDriver_verifyAuthenticationDriverLevel() {
 	handleError(err)
 }
 
+func ExampleConfig_tlsSelfSignedCertificates() {
+	// Create a certificate pool and add your CA certificate
+	certPool := x509.NewCertPool()
+	cert, err := os.ReadFile("path/to/ca.crt")
+	handleError(err)
+
+	// Add the CA certificate to the pool
+	certPool.AppendCertsFromPEM(cert)
+
+	// Create a driver with custom TLS configuration for self-signed certificates
+	driver, err := NewDriver(getUrl(), getAuth(), func(config *config.Config) {
+		config.TlsConfig = &tls.Config{
+			RootCAs:    certPool,
+			MinVersion: tls.VersionTLS12,
+		}
+	})
+	handleError(err)
+	defer handleClose(ctx, driver)
+
+	// Use the driver as normal - it will now trust your self-signed certificates
+	result, err := ExecuteQuery(ctx, driver, "RETURN 1 as num", nil, EagerResultTransformer)
```

Comment:
```suggestion
	result, err := ExecuteQuery(ctx, driver, "RETURN 1 AS num", nil, EagerResultTransformer)
```
nit: Cypher style guide :art: